### PR TITLE
customer scenario with puppet class BZ added

### PR DESCRIPTION
**Pull request by tstrych**
Asked by Vijay to implement this.

was raised by Support Team for test coverage because of CU's impacted

Test result. 
`========== 1 passed, 54 deselected, 21 warnings in 110.02s (0:01:50) ===========`
All warnings are connected to different problem `InsecureRequestWarning: Unverified HTTPS request is being made.`

### CODE REVIEW

File: tests/foreman/cli/test_host.py [hunk 1]
```diff
@@ -1004,6 +1004,42 @@ def test_hammer_host_info_output():
     assert int(result_info['additional-info']['owner-id']) == user.id
 
 
+@pytest.mark.host_create
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_host.py:1007]:
```suggestion
@pytest.mark.host_update
```

File: tests/foreman/cli/test_host.py [hunk 2]
```diff
@@ -1004,6 +1004,42 @@ def test_hammer_host_info_output():
...
+    """Update host owner and check puppet clases associated to the host
+
+    :id: 2b7dd148-914b-11eb-8a3a-98fa9b6ecd5a
+
+    :expectedresults: Host is updated with new owner
+        and has puppet class is still assigned and shown
+
+    :CaseImportance: Medium
+
+    :BZ: 1851149
```
Comment (Gauravtalreja1) [inline tests/foreman/cli/test_host.py:1021]: Include 6.8 BZ as well, which is the original one, and mark + there.
```suggestion
    :BZ: 1851149, 1809952
```

**Review by JacobCallahan — APPROVED** [review]
ACK pending @Gauravtalreja1's suggestions

File: tests/foreman/cli/test_host.py [hunk 3]
```diff
@@ -1004,6 +1004,42 @@ def test_hammer_host_info_output():
...
+@pytest.mark.tier2
+def test_positive_update_host_owner_and_verify_puppet_class_name(
+    module_env_search, module_org, module_location, module_puppet_classes, module_user
+):
+    """Update host owner and check puppet clases associated to the host
+
+    :id: 2b7dd148-914b-11eb-8a3a-98fa9b6ecd5a
+
+    :expectedresults: Host is updated with new owner
+        and has puppet class is still assigned and shown
```
Comment (swadeley) [inline tests/foreman/cli/test_host.py:1017]:
```suggestion
        and puppet class is still assigned and shown
```

File: tests/foreman/cli/test_host.py (hunk 2 above)
Comment (swadeley) [reply tests/foreman/cli/test_host.py:1021]: Please add UUID to BZ QA Whiteboard

Comment (tstrych) [reply tests/foreman/cli/test_host.py:1021]: Done :+1:

**Comment by tstrych** [comment]
trigger: test-robottelo
pytest: -k test_positive_update_host_owner_and_verify_puppet_class_name tests/foreman/cli/test_host.py

**Review by mshriver — APPROVED** [review]

**Comment by mshriver** [comment]
All outstanding comments have been addressed.

**Pull request merged**